runtime/doc/dev_theme.txt

```diff
@@ -49,8 +49,7 @@ Palettes
 - Each palette has the following colors (descriptions are for dark background;
   reverse for light one):
 
-    - Four shades of "colored" greys for general UI. In 256 colors they are
-      exact greys; in true colors they are shades of "cold" grey.
+    - Four shades of colored "cold" greys for general UI.
```

Comment:
Contender for the smallest nit picked: since we use AE ("colors"), "gray" is more common than "grey" (which is mostly BE).

Reply:
Hmm... Default colors are `Nvim...Grey`. Is it an issue? Other greys are doubled with "grey" and "gray", by the way.

Reply:
Not an issue; I just noticed the juxtaposition of "color" (AE) and "grey" (BE). As long as we are consistent, no need to support both spellings.

Reply:
As "grey" seems to be used more often than "gray" in help files (albeit pretty close), I'd stick with "grey".